tests/UsersTest.php

```diff
@@ -149,13 +149,17 @@ public function testNormalUserShouldNotBeAbleDisableMFA()
 
     public function testsRemoveUserFromEverywhere()
     {
-        $organization = $this->client->createOrganization($this->testMaintainerId, ['name' => 'ToRemoveOrg']);
-        $project = $this->client->createProject($organization['id'], ['name' => 'ToRemoveProj']);
+        $organization = $this->client->createOrganization($this->testMaintainerId, ['name' => 'ToRemoveOrg-1']);
```

Comment:
Když ti to tu failuje na duplicitě, tak to bude failovat i na travisu. Takže je potřeba po sobě ty organizace uklidit a zaházet.  Možná lepší je ty věci, co potřebuješ vytvořit, nejdřív smazat na začátku testu.

Reply:
Mně to nefailuje. Jenom jsem to přejmenoval:) Podle mě tam žádný check na duplicitu není.
![snimek z 2018-12-21 16-19-01](https://user-images.githubusercontent.com/5272798/50349466-2c3eca80-053c-11e9-9315-2d39d3a67f4b.png)

Každopádně to otevírá téma, který jsem chtěl nahodit! To je úklid po testech a testovací environment. Zkusím to napsat úplně ve zkratce. Testovací prostředí pro manage -> mysql v ram-disku -> import db skeletonu -> spuštění testů -> ?zahození databáze?

Reply:
Tohle tema si nech idealne na offsite

Reply:
Co se týče úklidu po testech, tak většinou se dělá před/po testech. https://github.com/keboola/storage-api-php-client/blob/593403cd24ed4b8fd60609c663ae57d33d55d436/tests/Backend/Workspaces/WorkspacesTestCase.php#L17

Reply:
uklízíme vždy před testama.

Reply:
Všechny organizace v testovacím maintainerovi se mažou na začátku testů

Reply:
To znamená, že nic řeším nemusím. Resolve!